### PR TITLE
Fixs text embedding translator regression

### PR DESCRIPTION
**Pull request by frankfliu**
## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

File: extensions/tokenizers/src/test/java/ai/djl/huggingface/tokenizers/TextEmbeddingTranslatorTest.java [hunk 1]
```diff
@@ -139,6 +156,27 @@ public void testTextEmbeddingTranslator()
...
+        // dense and layerNorm
+        criteria =
+                Criteria.builder()
+                        .setTypes(String.class, float[].class)
+                        .optModelPath(modelDir)
+                        .optBlock(block)
+                        .optEngine("PyTorch")
+                        .optArgument("tokenizer", "bert-base-uncased")
+                        .optArgument("dense", "linear.safetensors")
+                        .optArgument("denseActivation", "tahn")
```
Comment (siddvenk) [inline extensions/tokenizers/src/test/java/ai/djl/huggingface/tokenizers/TextEmbeddingTranslatorTest.java:168]: typo? should it be `Tanh`?

also, for some of these string comparisons do we want to compare the lower case versions?

File: extensions/tokenizers/src/main/java/ai/djl/huggingface/translator/TextEmbeddingTranslator.java [hunk 2]
```diff
@@ -160,18 +160,47 @@ static NDArray processEmbedding(
...
             default:
                 throw new AssertionError("Unexpected pooling mode: " + pooling);
         }
+
+        embedding = embedding.toDevice(Device.cpu(), false);
+        if (denseModel != null) {
+            NDArray weight = denseModel.get("linear.weight");
+            NDArray bias = denseModel.get("linear.bias");
+            embedding = embedding.getNDArrayInternal().linear(embedding, weight, bias).get(0);
+            if ("Tanh".equals(denseActivation)) {
```
Comment (siddvenk) [inline extensions/tokenizers/src/main/java/ai/djl/huggingface/translator/TextEmbeddingTranslator.java:186]: should this be `"tanh"`? do we want to compare lower case of denseActivation?

Comment (frankfliu) [reply extensions/tokenizers/src/main/java/ai/djl/huggingface/translator/TextEmbeddingTranslator.java]: Fixed

**Review by siddvenk — APPROVED** [review]
